src/lib/api/contrib/elasticsearch/ESResponseSerializer.js

```diff
@@ -16,10 +16,12 @@ export class ESResponseSerializer {
    * @param {object} payload the backend response payload
    */
   serialize(payload) {
+    const { aggregations, hits, ...extras } = payload;
     return {
-      aggregations: payload.aggregations || {},
-      hits: payload.hits.hits.map(hit => hit._source),
-      total: payload.hits.total.value,
+      aggregations: aggregations || {},
+      hits: hits.hits.map((hit) => hit._source),
+      total: hits.total.value,
+      extras: extras,
```

Comment:
this `extras` is really a black hole, an open door to anything :(

Reply:
This, I just did it for completeness to comply with the Invenio one...I am fine with omitting it :). The `extras` is not mandatory!